spec/controllers/spotlight/catalog_controller_spec.rb

```diff
@@ -111,6 +111,29 @@
         expect(json['docs'].first['title']).to eq 'KAART der REYZE van drie Schepen naar het ZUYDLAND in de Jaaren 1721 en 1722'
       end
     end
+
+    describe 'GET manifest' do
+      context 'document is an uploaded resource' do
+        it 'returns the json manifest produced by Spotlight::IiifManifestPresenter, based on the retrieved document and the controller' do
+          allow_any_instance_of(::SolrDocument).to receive(:uploaded_resource?).and_return(true)
+          response_hash = { field: 'value' }
+          iiif_manifest_presenter = double(Spotlight::IiifManifestPresenter, iiif_manifest_json: response_hash.to_json)
+          expect(Spotlight::IiifManifestPresenter).to receive(:new).with(an_instance_of(::SolrDocument), controller).and_return(iiif_manifest_presenter)
+
+          get :manifest, params: { exhibit_id: exhibit, id: '1-1' }
```

Comment:
I was able to reproduce this locally.  I believe that in the context of this test (and on a fresh index) there is no resource w/ the id `1-1`.  You probably have one locally, which is why this passes for you.

What we probably want to do here is instead of using a hard-coded ID, we should use the `uploaded_resource` FactoryGirl factory to build a resource, and then pass that ID here to the `manifest` action.